api/src/pcapi/core/offers/offer_metadata.py

```diff
@@ -60,6 +61,11 @@ def _get_event_metadata_from_offer(offer: offers_models.Offer) -> Metadata:
         firstBeginningDatetime: datetime.datetime = offer.firstBeginningDatetime
         event_metadata["startDate"] = firstBeginningDatetime.isoformat(timespec="minutes")
 
+    event_metadata["eventAttendanceMode"] = "OnlineEventAttendanceMode" if offer.isDigital else "OfflineEventAttendanceMode"
+
+    if offer.extraData and offer.extraData["releaseDate"]:
```

Comment:
```suggestion
    if offer.extraData and "releaseDate" in offer.extraData:
```
pas tous les évènements ont une date de parution, ce qui fait planter lorsqu'on appelle `offer.extraData["releaseDate"]`, c'est un comportement différent entre js et python :

```
➜ node
Welcome to Node.js v18.12.1.
Type ".help" for more information.
> {}['2']
undefined

➜ python
Python 3.11.5 (main, Sep  2 2023, 14:16:33) [GCC 13.2.1 20230801] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> {}['2']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: '2'
```

Reply:
Yes ! Avec l'approval de Corentin j'ai utilisé la méthode .get() des dictionnaires, c'est ok aussi ? 🙂